core-api/src/main/java/com/optimizely/ab/config/audience/UserAttribute.java

```diff
@@ -118,20 +118,35 @@ public Boolean evaluate(ProjectConfig config, Map<String, ?> attributes) {
         return null;
     }
 
-    @Override
-    public String toString() {
+    public String getValueStr() {
         final String valueStr;
         if (value == null) {
             valueStr = "null";
         } else if (value instanceof String) {
-            valueStr = String.format("'%s'", value);
+            valueStr = String.format("%s", value);
         } else {
             valueStr = value.toString();
         }
+        return valueStr;
+    }
+
+    @Override
+    public String toJson() {
+        StringBuilder attributes = new StringBuilder();
+        if (name != null) attributes.append("{\"name\":\"" + name + "\"");
+        if (type != null) attributes.append(", \"type\":\"" + type + "\"");
+        if (match != null) attributes.append(", \"match\":\"" + match + "\"");
+        attributes.append(", \"value\":" + ((value instanceof String) ? ("\"" + getValueStr() + "\"") : getValueStr()) + "}");
+
+        return attributes.toString();
+    }
+
+    @Override
+    public String toString() {
         return "{name='" + name + "\'" +
             ", type='" + type + "\'" +
             ", match='" + match + "\'" +
-            ", value=" + valueStr +
+            ", value=" + ((value instanceof String) ? ("'" + getValueStr() + "'") : getValueStr()) +
```

Comment:
why single quote here?

Reply:
This was part of the original logic that I didn't want to change (Line 127)  but don't need for the function above and wanted to reuse the same code minus those quotes. We can chat offline if you like.